linalg/ode.hpp

```diff
@@ -12,20 +12,59 @@
 #ifndef MFEM_ODE
 #define MFEM_ODE
 
+#include "../general/communication.hpp"
 #include "../config/config.hpp"
 #include "operator.hpp"
-#include "../general/communication.hpp"
+#include <vector>
+
 
 namespace mfem
 {
 
+/// A class for storing states of previous timesteps
+class StateData
```

Comment:
This class name is a little bit general. Would `MultistepODEStateData` or something similar convey the purpose?

Reply:
Good suggestion, however,  I don't think generalized alpha counts as a multistep method.
How about `ODEStateData`?

Reply:
Sounds good to me 👍